step.yml

```diff
@@ -72,17 +70,25 @@ inputs:
         Example: `file://./BuildAnything.mobileprovision`
   - keychain_path: $HOME/Library/Keychains/login.keychain
     opts:
-      category: Config
       title: Keychain Path
       is_required: true
   - keychain_password: $BITRISE_KEYCHAIN_PASSWORD
     opts:
-      category: Config
       title: Keychain's password
       is_required: true
+  - install_defaults: "Yes"
+    opts:
+      category: Default Codesign Files
+      summary: Should install the default Codesign Files
+      title: Install default codesign files
+      description: |
+        You can turn off the default codesign files install.
```

Comment:
Please describe just a little bit more. For example what is default codesign files? The user will see Bitrise related provision profiles and certificates listed in steps. Please try to focus on what the user can relate on and match the description/title/summary to that.